tox.ini

```diff
@@ -182,3 +185,30 @@ deps=
     psycopg2
     djangorestframework
     unittest2
+
+[testenv:py34-django20]
```

Comment:
Since 1.9 was not a real release and just a 2.0 pre-release, we can probably remove it.

Reply:
You're saying remove all of the 1.9 tests?

Reply:
Ah, I was misremembering. I thought it was the pre-release